lib/storage/tests/integration/alias_tests.rs

```diff
@@ -55,6 +55,8 @@ fn test_alias_operation() {
         handle_collection_load_errors: false,
         recovery_mode: None,
         async_scorer: false,
+        update_concurrency: Some(NonZeroUsize::new(2).unwrap()),
```

Comment:
I don't think this should get in as there is no assertion associated with it.

Reply:
Though I have no clear idea on how to test this then, I do think it's good that we at least cover it somewhere.

Any ideas on a test?